tip-wlan/charts/wlan-integrated-cloud-component-service/templates/secret.yaml

```diff
@@ -22,9 +22,7 @@ metadata:
     heritage: {{ .Release.Service | quote }}
 type: Opaque
 data:
-  {{- if .Values.creds.enabled }}
-  postgresql-password: {{ .Values.creds.postgres.password | b64enc | quote }}
-  schema-repo-user: {{ .Values.creds.schema_repo.username | b64enc | quote }}
-  schema-repo-password: {{ .Values.creds.schema_repo.password | b64enc | quote }}
-  {{- end }}
+  postgresql-password: {{ .Values.global.postgres.password | b64enc | quote }}
```

Comment:
Why have we removed the `if ... creds.enabled` clause?

Reply:
yeah, mentioned this in the pr notes, they don't make any sense as the service won't work without these

Reply:
@4c74356b41 as we discussed a few times in our weekly calls, Max is about to work on a feature that will allow us to run the backend without TLS... So the service will work without creds.

Reply:
took this offline with max, agreed that this is fine for now